pkg/resources/sop_url.go

```diff
@@ -21,6 +21,7 @@ const (
 	SopUrlAlertsAndTroubleshooting                            = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md"
 	sopUrlCloudResourceDeletionStatusFailed                   = "https://github.com/RHCloudServices/integreatly-help/tree/master/sops/2.x/alerts/clean_up_cloud_resources_failed_teardown.asciidoc"
 	sopUrlSendGridSmtpSecretExists                            = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/sendgrid_smtp_secret_not_present.asciidoc"
+	sopUrlDeadMansSnitchSecretExists                          = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/deadmanssnitch_secret_not_present.asciidoc"
```

Comment:
Should the path to this file be:
```suggestion
	sopUrlDeadMansSnitchSecretExists                          = "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/rhoam/alerts/deadmanssnitch_secret_not_present.asciidoc"
```
Not a big thing and I know it would mean changing the location in the help repo also.

Small thing, I don't have a strong opinion on it.

Reply:
Done. Alert moved to `sops/rhoam/alerts` and renamed according to folder convention.
Thank you